index.html

```diff
@@ -436,30 +451,39 @@ <h2><dfn>disconnect()</dfn> method</h2>
     <h2>Processing</h2>
     <section data-link-for="PerformanceObserver">
     <h2>Queue a <code>PerformanceEntry</code></h2>
-      <p>To <dfn>queue a PerformanceEntry</dfn> (<var>new entry</var>) with an
-      optional <var>add to performance entry buffer</var> flag, which is unset
-      by default, run these steps:</p>
+      <p><i>Specifications</i> of performance entries may define an algorithm
+        <var>should append to observer</var> which receives a <a>PerformanceEntry</a> and
+        a <a>PerformanceObserver</a> as input and returns a boolean specifying whether the
+        observer should be notified of the entry.
+      <p>The <dfn>queue a PerformanceEntry</dfn> <var>new entry</var> with an optional
+        <var>add to performance entry buffer</var> flag, which is unset by default, run
+        these steps:</p>
     <ol>
-      <li>Let <i>interested observers</i> be an initially empty set of
+      <li>Let <var>interested observers</var> be an initially empty set of
       <a>PerformanceObserver</a> objects.
       </li>
-      <li>For each <a>registered performance observer</a> (<i>observer</i>):
+      <li>For each <a>registered performance observer</a> (<var>observer</var>), append
+        <var>observer</var> to <var>interested observers</var> if all of the following
+        conditions are met:
         <ol>
-          <li>If <i>observer</i>'s <a>PerformanceObserverInit</a> <a data-lt=
-          "PerformanceObserverInit.entryTypes">entryTypes</a> includes <i>new
-          entry</i>’s <a data-lt="PerformanceEntry.entryType">entryType</a>
-          value, append <i>observer</i> to <i>interested observers</i>.
+          <li><var>new entry</var>’s <a data-lt="PerformanceEntry.entryType">entryType</a>
+          value is in <var>observer</var>'s <a>PerformanceObserverInit</a>: either its
+          <a data-lt="PerformanceObserverInit.entryTypes">entryTypes</a> includes it or it
+          has a dictionary with that name.</li>
+          <li><var>should append to observer</var>(<var>new entry</var>,
+            <var>observer</var>) returns true, if defined by the <i>specification</i> of
```

Comment:
For cloning steps, I thought there was an inheritance like approach taken, where specs could override the default. That feels a bit slicker than what we've got here.

Reply:
Hm no, I don't think there is a default. The closure approach has a default, the cloning steps approach says that a specification may specify additional steps that should be run at a certain point in time.

Reply:
You're right!